.circleci/config.yml

```diff
@@ -221,6 +221,8 @@ commands:
       - run: cargo sweep -s
       - install_extra_tools:
           os: << parameters.os >>
+      # cargo-deny fetches a rustsec advisory DB, which has to happen on github.com over https
+      - run: git config --global --unset-all url.ssh://git@github.com.insteadof
```

Comment:
this seems like it would cause other problems.  like when we want to fetch things that are on a private repo using a circleci / git deploy key or something?

i'm surprised as to why that git remote for the advisories couldn't be fetched over gitssh?  it's a public repo and i'd expect it to work?

Reply:
could the override be only for the advisories repo?